src/components/application_manager/src/commands/hmi/get_urls.cc

```diff
@@ -93,7 +93,6 @@ void GetUrls::ProcessPolicyServiceURLs(const policy::EndpointUrls& endpoints) {
 
   if (!app_id_to_send_to) {
     SDL_ERROR("There are no available applications for processing.");
```

Comment:
@dev-gh I've created question in order to clarify it. Will fix it asap I have the answer.
[APPLINK-27873](https://adc.luxoft.com/jira/browse/APPLINK-27873)